Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN apt-get update && apt-get install -y \
 ADD . /code/wasm-c-api
 WORKDIR /code/wasm-c-api
 RUN make v8-checkout && make -j v8
-RUN make C_COMP=gcc LD_FLAGS= C_FLAGS= 
+RUN make clean && make C_COMP=gcc LD_FLAGS= C_FLAGS="-fPIC"
```

Comment:
Why do you need to clean here?

Reply:
By the time we get here `wasm-bin.o` is already built without `-fPIC` and that was causing a failure. Not sure what builds it though. Got to investigate.

Reply:
That is very odd. Have you checked that your build dir is clean? I saw similar problems when I still had an out dir lying around when running docker.